shell/src/main/java/alluxio/shell/command/CopyFromLocalCommand.java

```diff
@@ -91,6 +91,11 @@ private void copyFromLocalDir(File srcDir, AlluxioURI dstPath)
     createDstDir(dstPath);
     List<String> errorMessages = new ArrayList<>();
     File[] fileList = srcDir.listFiles();
+    if (fileList == null) {
+      String errMsg = String.format("copyFromLocal %s %s failed!",
+              srcDir.toString(), dstPath.toString());
+      throw new IOException(errMsg);
```

Comment:
@goodluckbot to avoid the NPE we could set fileList to an empty array after storing the error message. We need to do cleanup because we've already called `createDstDir(dstPath)` to create the destination directory